client/db_test.go

```diff
@@ -38,55 +38,104 @@ func ExampleDB_Get() {
 	s, db := setup()
 	defer s.Stop()
 
-	fmt.Println(db.Get("aa").String())
+	result := db.Get("aa")
+	if result.Err != nil {
+		panic(result.Err)
+	}
+	fmt.Printf("aa=%s\n", result.Rows[0].ValueBytes())
```

Comment:
You can, but real code wouldn't. Real code would want the value of the "aa" record, not a string like "0: aa=nil".